src/dispatch/incident/flows.py

```diff
@@ -676,27 +614,7 @@ def incident_closed_flow(incident_id: int, command: Optional[dict] = None, db_se
     """Runs the incident closed flow."""
     # we load the incident instance
     incident = incident_service.get(db_session=db_session, incident_id=incident_id)
-
-    if incident.status == IncidentStatus.active:
-        # we run the stable flow and let the user know
-        if command:
-            convo_plugin = plugins.get(INCIDENT_PLUGIN_CONVERSATION_SLUG)
-            convo_plugin.send_ephemeral(
-                command["channel_id"],
-                command["user_id"],
-                "Mark Incident Stable Notification",
-                blocks=[
-                    {
-                        "type": "section",
-                        "text": {"type": "plain_text", "text": "Marking the incident as stable..."},
-                    }
-                ],
-            )
-        incident_stable_flow(incident_id, command=command, db_session=db_session)
```

Comment:
Same as active and stable flows, plus we need to run the stable flow if incident status is active.

Reply:
Same as others stable flow should run if `update_flow` thinks it needs to.